tests/data/format/numpydoc/numpydoc_header_line.py.out

```diff
@@ -0,0 +1,63 @@
+"""Example module for numpydoc docstring style.
+References
+----------
+NumPy docstring style guide:
+https://numpydoc.readthedocs.io/en/latest/format.html#documenting-modules"""
+import math
+
+EULER_NUMBER = math.e
+"""Euler's number.
+
+Not related to Euler's constant (sometimes called the Euler-Mascheroni
+constant.
+References
+----------
+E (mathematical constant)
+https://en.wikipedia.org/wiki/E_(mathematical_constant)
+Notes
+-----
+It is the limit of ``(1 + 1/n)**n`` as n approaches infinity, so it
+is used in the equation for continuously-compouned interest.
+
+It is also the sum of the reciprocals of the whole numbers starting
+with zero, which is related to some calculus-related properties
+mathemeticians find elegant.
+"""
+
+
+def sincos(theta):
+    """Returns
+    -------
```

Comment:
This doesn't seem correct? Shouldn't it be a little longer?

Reply:
It matches the length of the word "Returns", and is shorter in the file by three characters, matching the three double quotes starting that line.

It might be clearer as
```python
def sincos(theta):
    """\
    Returns
    -------
    ...
    """
```
(same number of hyphens)

Reply:
I mean, there isn't really a style guide for this I think, but imo it makes sense to add more `-` here so that the second line covers both the quotes and `Returns`.